Havit/Havit/Network/API/SearchContentsEndPoint.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SearchContentsEndPoint.swift
+//  Havit
+//
+//  Created by 박예빈 on 2022/01/19.
+//
+
+import Foundation
+
+enum SearchContentsEndPoint {
+    case getSearchResult(keyword: String)
+    
+    var requestTimeOut: Float {
+        return 20
+    }
+
+    var httpMethod: HttpMethod {
+        return .GET
+    }
+
+    var requestBody: Data? {
+        return nil
+    }
+
+    func getURL(from environment: APIEnvironment) -> String {
+        let baseUrl = environment.baseUrl
+        switch self {
+        case .getSearchResult(let keyword):
+            let urlString = "\(baseUrl)/content/search/?keyword=\(keyword)"
+            return urlString
```

Comment:
p1;

```suggestion
      return "\(baseUrl)/content/search/?keyword=\(keyword)"
```
이렇게 써도 될 거 같아요.

Reply:
87dcee00389